packages/api/src/auth/decoders/clerk.ts

```diff
@@ -1,38 +1,28 @@
-import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
+import type IClerk from '@clerk/clerk-sdk-node/instance'
 
-interface Req {
-  event: APIGatewayProxyEvent
-  context: LambdaContext
-}
-
-export const clerk = async (token: string, req: Req) => {
+export const clerk = async (token: string) => {
   // Use require here, to prevent needing clerk sdk in api deps
-  const { sessions, users } = require('@clerk/clerk-sdk-node')
+  const Clerk = require('@clerk/clerk-sdk-node/instance').default
+
+  const { users, base }: IClerk = new Clerk()
 
   if (!process.env.CLERK_API_KEY) {
     console.error('CLERK_API_KEY env var is not set.')
     throw new Error('CLERK_API_KEY env var is not set.')
   }
 
-  // Clerk sessions are a combination of a clerk "current session id", which we store
-  // in the Redwood auth token, and the __session cookie, which contains a second session
-  // bearer token. The two tokens together define which device is browsing and as who.
-  const clerkCookieName = '__session'
-  const cookies = req.event.headers['cookie']?.split(';').map((c) => c.trim())
-  const sessionCookie = cookies
-    ?.find((c) => c.startsWith(clerkCookieName + '='))
-    ?.substring(clerkCookieName.length + 1)
-
-  if (!sessionCookie || sessionCookie.length < 1) {
-    return Promise.reject(new Error('Clerk __session token is not set'))
+  if (!process.env.CLERK_JWT_KEY) {
+    console.error('CLERK_JWT_KEY env var is not set.')
+    throw new Error('CLERK_JWT_KEY env var is not set.')
   }
 
-  const session = await sessions.verifySession(token, sessionCookie)
-  if (!session.userId) {
+  const jwtPayload = await base.verifySessionToken(token)
+
+  if (!jwtPayload.sub) {
     return Promise.reject(new Error('Session invalid'))
   }
 
-  const user = await users.getUser(session.userId)
+  const user = await users.getUser(jwtPayload.sub)
```

Comment:
Which values do Redwood developers critically expect on the user object?  One day we'd like to build them into `jwtPayload` so we can avoid this network request, too.

(Currently we don't have a way to add publicMetadata to the default payment, so that alone makes this an issue we won't solve for this PR. Just interested for the future)

Reply:
@dthyresson can you help with an answer here, please?

My expectations would be:
* roles
* user id

Anything else can always be fetched by the RW developer by a manual call to some `getUser` equivalent. But, as you say, the more you can include, the less likely it will be that the developer has to do that extra api roundtrip.

Reply:
We need: id aka sub, roles (at root or within appMetadata).

People will want email and some profile info: photo, name (can be part of userMetadata).

Reply:
Thank you! We'll reference this during our next jwt revision.